### PR TITLE
add fruit package

### PR DESCRIPTION
**Pull request by arrtchiu**

### CODE REVIEW

**Comment by ruslo** [comment]
Please add an example so I can run CI tests.

CURL example:
* https://github.com/ruslo/hunter/tree/master/examples/CURL

**Comment by arrtchiu** [comment]
Thanks Ruslo, have got part the way there but stuck at getting Fruit building. Hitting this error: https://github.com/google/fruit/blob/master/CMakeLists.txt#L11-L13

```
if(NOT "${CMAKE_BUILD_TYPE}" MATCHES "^(Debug|Release|RelWithDebInfo|MinSizeRel)$")
  message(FATAL_ERROR "Please re-run CMake, specifying -DCMAKE_BUILD_TYPE=Debug , -DCMAKE_BUILD_TYPE=Release , -DCMAKE_BUILD_TYPE=RelWithDebInfo or -DCMAKE_BUILD_TYPE=MinSizeRel .")
endif()
```

What's the best way for passing on `CMAKE_BUILD_TYPE` to the package?

**Comment by ruslo** [comment]
tl;dr it's a error in CMakeLists.txt of google/fruit

I guess this is because you're using multi-config generator (Xcode or Visual Studio). For such generators it make no sense to set `CMAKE_BUILD_TYPE` so Hunter do not set this variable. Take a look here for details:
* https://stackoverflow.com/a/24470998/2288008

**Comment by ruslo** [comment]
Closing inactive pull request.

**Comment by FelikZ** [comment]
I have adopted pull request in my fork and it works for me as expected.

At the moment no time to make proper PR, so if somebody will find it using search engine, have a look at those:
https://github.com/FelikZ/hunter/tree/add-fruit
https://github.com/FelikZ/fruit/tree/hunter-package
